aspnetcore/fundamentals/localization/sample/Resources/Views/Book/Index.resx

```diff
@@ -198,8 +198,8 @@
   <data name="Run &amp; Deploy" xml:space="preserve">
     <value>Déployer &amp; Exécuter</value>
   </data>
-  <data name="Run commands in your project.json" xml:space="preserve">
-    <value>Exécuter des commandes dans votre project.json</value>
+  <data name="Run commands in your bash" xml:space="preserve">
```

Comment:
What does "Run commands in your project.json" doesn't make sense to me and "bash" doesn't sound like a direct replacement for "project.json".  Can you explain what's going on here?